Elf/elf_lookup.py

```diff
@@ -615,6 +621,20 @@
     STT_HIPROC:"End of processor-specific",
 }
 
+STV_DEFAULT     = 0
+STV_INTERNAL    = 1
+STV_HIDDEN      = 2
+STV_PROTECTED   = 3
+
+stv_lookup ={y: x for x, y in globals().items() if x.startswith("STV_")}
+
+st_other_visibility = {
+    STV_DEFAULT:"Symbol visibility specified by binding type",
+    STV_INTERNAL:"Symbol visibility is reserved",
+    STV_HIDDEN:"Symbol is not visible to other objects",
+    STV_PROTECTED:"Symbol is visibl by other objects, but cannot be preempted"
```

Comment:
missing an e :)

Reply:
the 'e' has been found and returned to it's rightful place.  i was sipping margaritas in Tahiti.